frontend/javascripts/oxalis/view/rendering_utils.ts

```diff
@@ -106,6 +117,26 @@ export async function downloadScreenshot() {
     const buffer = renderToTexture(planeId, null, null, false, clearColor);
 
     const inputCatcherElement = document.querySelector(`#inputcatcher_${planeId}`);
+    const scalebar = inputCatcherElement?.querySelector(".scalebar") as HTMLElement;
+    let drawImageIntoCanvasCallback = null;
+    if (scalebar) {
+      drawImageIntoCanvasCallback = (ctx: CanvasRenderingContext2D) => {
+        const scalebarDistanceToRightBorder =
+          scalebar.offsetParent != null
+            ? scalebar.offsetParent.clientWidth - scalebar.offsetLeft - scalebar.clientWidth
+            : 0;
+        const scalebarDistanceToTopBorder = scalebar.offsetTop;
```

Comment:
Nitpicky, but I was wondering why the top of the logo and scale bar don't exactly align and I think it's caused by the following: `offsetTop` is relative to the parent's [content-box](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#content-box), but the inputcatchers use [box-sizing: border-box](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing#border-box) with a 2 px wide border (`OUTER_CSS_BORDER`). These 2 px need to be added here, to be exact :)